/*
rethrow-outside-catch.js(4,1): BCE0081: A throw statement with no arguments is not allowed outside an exception handler.
*/
throw;
